core/store/models/eth.go

```diff
@@ -22,10 +22,20 @@ type EthTxMeta struct {
 	TaskRunID uuid.UUID
 }
 
+func (EthTxMeta) GormDataType() string {
```

Comment:
Now that the v1 pipeline is gone we can remove this EthTxMeta and rename EthTxMetaV2 to EthTxMeta. Also I think EthTxMetaV2 would better belong in bptxm/models.go